week-1-practise/level-1/8-function-output-1.js

```diff
@@ -6,7 +6,7 @@ var result = getNumber();
 console.log(result);
 
 // Answer these questions:
-// - What does this program do?
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+// - What does this program do? it returns a random number and multiplies it by 10.
+// - How many parameters does the function take? 1
```

Comment:
Are you sure? 🤔 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/length